### DISCUSSION TITLE
How to find minimal version for an option?

### DISCUSSION THREAD

**Discussion by VladimirSlavik** [containers/podman#19773] [Q&A] [answered]
How do I find what's the version needed for an option?

I run `podman build --cache-ttl=***` locally with 4.6.0, but it does not work for me in Github actions. The podman there is supposedly 3.4.4. So I tried finding this in the changelog... and got nothing useful for ttl, cache, live... Is there some other resource to check?

**Comment by vrothberg** [comment]
Thanks for reaching out!  Usually the changelog is the right resource to consult for finding out which version a feature got shipped with. In this case, I assume an oversight and we missed adding it.

@flouthoc, do you know when this flag was added to Podman?

**Reply by VladimirSlavik** [reply]
Thank you, I guessed it was a mistake, which can't be fixed retroactively of course...

**Comment by flouthoc** [comment, marked as answer]
@vrothberg The feature which @VladimirSlavik is looking for was added for build-cache ( https://github.com/containers/buildah/pull/4165 ) in buildah `v1.27.0` here: https://github.com/containers/buildah/releases/tag/v1.27.0 so it should be available in podman after and from `v4.2.0`
